portal/server/controllers/dashboard/facilities/template-filters.ff-test.js

```diff
@@ -0,0 +1,36 @@
+import { FACILITY_STATUS } from '@ukef/dtfs2-common';
+import { stageFilters } from './template-filters';
+import { FIELD_NAMES } from '../../../constants';
+import { DASHBOARD_FILTERS } from '../../../content-strings';
+import { generateFiltersArray } from '../filters/generate-template-filters';
+
+describe('controllers/dashboard/facilities - template-filters', () => {
+  describe('stageFilters - ff enabled', () => {
```

Comment:
Can we say "feature flag" instead of `ff` please

Not for this PR but I think in all feature flag related tests/descriptions/documentation, we need to mention explicitly _which_ feature flag it is.

E.g looking at this, I have no idea which feature flag affects this functionality

Reply:
In other tests we have this 

```
describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to true', () => {
```

Reply:
Seems the existing feature flag tests/descriptions are mixed and matched 😞

Reply:
I think `describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to true', () => {` is from before Abhi introduced the `ff-test` tests, they typically contain a `beforeEach` setting the FF to true